fem/quadinterpolator.cpp

```diff
@@ -540,34 +541,58 @@ void QuadratureInterpolator::Mult(const Vector &e_vec,
    MFEM_ASSERT(fespace->GetMesh()->GetNumGeometries(
                   fespace->GetMesh()->Dimension()) == 1,
                "mixed meshes are not supported");
+   MFEM_ASSERT(useAbs?use_tensor_eval:true,
+               "AbsMult only implemented for tensor elements!")
+
+   DofToQuad abs_maps;
+   if (useAbs)
+   {
+      abs_maps.FE = maps.FE;
+      abs_maps.IntRule = maps.IntRule;
+      abs_maps.mode = maps.mode;
+      abs_maps.ndof = maps.ndof;
+      abs_maps.nqpt = maps.nqpt;
+      abs_maps.B = maps.B;
+      abs_maps.Bt = maps.Bt;
+      abs_maps.G = maps.G;
+      abs_maps.Gt = maps.Gt;
```

Comment:
I think `abs_maps = maps;` will work here instead

Reply:
Since similar logic is performed in the integrators, we might want
`DofToQuad DofToQuad::Abs() const` that does this.